### ISSUE TITLE
How do you use this?

### ISSUE THREAD

**Issue by alicchen** [Airmail/Mailchimp-AMPlug#4]
I can't seem to figure how to use the plugin. I have it installed and activated, but there doesn't seem to be a menu item to allow me to add contacts to Mailchimp. I also couldn't find any documentation about this plugin. Any help would be appreciated.

**Comment by samgoeta** [comment]
Yes there is not any documentation and the plugin does not seem to achieve anything 🤔

**Comment by KodaKoder** [comment]
right click on a message in the menu should be an add to mailchimp actions

**Comment by samgoeta** [comment]
Which I did with all my emails. Which sent about 10 emails to each of my contact. 😱
